presto-main/src/main/java/io/prestosql/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -308,6 +309,12 @@ protected Boolean visitInPredicate(InPredicate node, Void context)
             return process(node.getValue(), context) && process(node.getValueList(), context);
         }
 
+        @Override
+        protected Boolean visitFormat(Format node, Void context)
+        {
+            return node.getArguments().subList(1, node.getArguments().size()).stream().allMatch(expression -> process(expression, context));
```

Comment:
This needs to process *all* arguments to `Format`. Otherwise, the analyzer won't realize this is an invalid query:

```
SELECT format(s, sum(v)) FROM (VALUES ('%s', 1)) t(s, v)
```

(`s` must be in the group by clause)

Reply:
ah...thanks, will fix the code and squash the commits